composer.json

```diff
@@ -1,12 +1,15 @@
 {
-    "name": "silverstripe/staticpublishqueue",
+    "name": "sunnysideup/staticpublishqueue",
```

Comment:
We can't change this

Reply:
;-) Sorry, I wanted to use this new version so I changed it temporarily so that I could publish on packagist.